### ISSUE TITLE
Some error when i run the code

### ISSUE THREAD

**Issue by csas-bowen** [awreed/CSAS_Deconvolution_INR#1]
When I run these two pieces of code, I will get the following two errors respectively. How should I solve it?
![error2](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/9014aa8e-3e9f-43e3-b694-95f196d52b1f)

![error1](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/c5142ae9-d6c3-4df1-bd95-ec1022c2da2e)

**Comment by awreed** [comment]
Hi,

For the first error, you can just comment out the Line 160 of run_all_methods (i.e., comment out save_sas_plot(task['gt'], os.path.join(meth_save_dir, 'gt.png'). There is no ground truth image for the airsas reconstructions since it is real data. 

For the second error, I think this is a version mismatch between our codes. It seems structural_similarity wants a data_range parameter (like in this example https://scikit-image.org/docs/stable/auto_examples/transform/plot_ssim.html)

**Comment by csas-bowen** [comment]
I really appreciate your personal guidance and help, with which I managed to run through the code. In fact, I am a beginner in CSAS, and now I have some questions about the experimental results that I want to communicate and discuss with you and I look forward to your guidance.
    Various deconvolution methods give the same results as in your post when running the airsas data with a bandwidth of 20kHz. However, when running the 5kHz airsas data, the deconvolution result of the sinr method is not normal, while other methods are normal.
![5K-deconv](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/638a162d-394a-476d-bafb-7398f11711da)
![5k-das](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/da5c0e15-4d5f-4630-88ff-c616fa910325)
The inr parameter settings for Airsas are as follows：
![parameter](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/1ecacb41-ed81-4479-9a61-f42c3117ca7f)

I noticed that the sinr deconvolution result of SLc13 you provided in the code shows that the result keeps improving as the epoh increases，but when i conducting SLc13 simulation, the deconvolution results of the SINR method will first increase and then decrease with the increase of epoh, with psnr and ssim showing the best results around epoh=1100, and there will be errors in the deconvolution results later on.
![SINR-EPOH1100](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/c0aa2624-f1d6-4e43-ae0c-c715983342ab)
![DAS-EPOH1100](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/dcdb7d2d-baeb-44cb-89a9-7f124e221937)
![SINR-EPOH1999](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/74136684-0cc0-4707-943c-cdd1f530d828)
![DAS-EPOH1999](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/490e7051-adce-4756-a133-851b8f5beae7)
The inr parameter settings for slc13 are as follows：
![UD_ }C_ @C~VSXOBE4@2(G7](https://github.com/awreed/CSAS_Deconvolution_INR/assets/132887867/d031c483-04d8-4f9c-b434-0ef4b14f0349)

**Comment by awreed** [comment]
Yes the results sometimes become worse at higher iterations if the network starts to overfit. In those cases you can use early stopping. 

I am not positive why the 5kHz AirSAS result isn't working. Make sure you are setting the scene_dir and bg_dir correctly:

```
SIZE_W, SIZE_H = 400, 400
# AirSAS measurements to be reconstructed
scene_dir = 'airsas_data/small_features_cutout_5k'
# AirSAS measurements of empty scene
bg_dir = 'airsas_data/bg_5k'

```

If that doesn't help, set SAVE_EVERY=5 and send me the results so I can look. You can email me at awreed@asu.edu

**Comment by awreed** [comment]
Also you may be interested in our follow up work:

https://github.com/awreed/Neural-Volumetric-Reconstruction-for-Coherent-SAS
